config/dosarray_config.sh

```diff
@@ -34,12 +35,17 @@ then
 fi
 
 function dosarray_physical_hosts_skip () {
-  local SKIP="$1"
-  echo `seq ${SKIP} $(( ${#DOSARRAY_PHYSICAL_HOSTS_PUB[@]} - 1 ))`
+  if [ ${DOSARRAY_TARGET_SERVER_INDEX} -eq 0 ]
+  then
+    echo `seq $(( ${DOSARRAY_TARGET_SERVER_INDEX} + 1)) $(( ${#DOSARRAY_PHYSICAL_HOSTS_PUB[@]} - 1 ))`
+  else
+    index=(`seq 0 $(( ${DOSARRAY_TARGET_SERVER_INDEX} - 1))` `seq $(( ${DOSARRAY_TARGET_SERVER_INDEX} + 1)) $(( ${#DOSARRAY_PHYSICAL_HOSTS_PUB[@]} - 1 ))`)
```

Comment:
Does this also work if `${DOSARRAY_TARGET_SERVER_INDEX} -eq (${#DOSARRAY_PHYSICAL_HOSTS_PUB[@]} - 1)`? (i.e., if `DOSARRAY_TARGET_SERVER_INDEX` is the last server)

Reply:
No it wouldn't. Will add the necessary condition